pyproject.toml

```
@@ -9,7 +9,7 @@ repository  = "https://github.com/nat-n/poethepoet"
 homepage    = "https://github.com/nat-n/poethepoet"
 
 [tool.poetry.dependencies]
-python = ">=3.7"
+python = "^3.7"
```

See comment on other PR
```suggestion
python = ">=3.7"
```